pandas/indexes/multi.py

```diff
@@ -856,6 +856,7 @@ def _get_level_values(self, level):
         Parameters
         ----------
         level : int level
+        copy  : bool whether copy of results should be done
```

Comment:
@jreback What I needed `_get_level_values()` to do is have the same behavior as the public `get_level_values()`,  with the assumption of the int argument, so the `copy` argument makes that happen by doing the shallow copy there.

When I first looked at this, there was no `_get_level_values()`, but that got introduced within the past 2 weeks, so I then had to make everything compatible.